node/src/metrics.rs

```diff
@@ -1,226 +1,201 @@
+});
 
-lazy_static! {
-    pub static ref MPC_NUM_SIGNATURE_COMPUTATIONS_LED: prometheus::IntCounterVec =
+pub static MPC_NUM_SIGN_RESPONSES_INDEXED: LazyLock<prometheus::IntCounter> = LazyLock::new(|| {
+    prometheus::register_int_counter!(
+        "mpc_num_signature_responses_indexed",
+        "Number of signatures responses seen by the indexer"
+    )
+    .unwrap()
+});
+pub static MPC_NUM_SIGNATURE_COMPUTATIONS_LED: LazyLock<prometheus::IntCounterVec> =
+    LazyLock::new(|| {
         prometheus::register_int_counter_vec!(
             "mpc_num_signature_computations_led",
             "Number of signature computations that this node led",
             &["result"],
         )
-        .unwrap();
-}
-
-lazy_static! {
-    pub static ref MPC_NUM_PASSIVE_SIGN_REQUESTS_RECEIVED: prometheus::IntCounter =
+        .unwrap()
+    });
+pub static MPC_NUM_PASSIVE_SIGN_REQUESTS_RECEIVED: LazyLock<prometheus::IntCounter> =
+    LazyLock::new(|| {
         prometheus::register_int_counter!(
             "mpc_num_passive_signature_requests_received",
             "Number of passive signature requests received from mpc peers"
         )
-        .unwrap();
-}
+        .unwrap()
+    });
 
-lazy_static! {
-    pub static ref MPC_NUM_PASSIVE_SIGN_REQUESTS_LOOKUP_SUCCEEDED: prometheus::IntCounter =
+pub static MPC_NUM_PASSIVE_SIGN_REQUESTS_LOOKUP_SUCCEEDED: LazyLock<prometheus::IntCounter> =
+    LazyLock::new(|| {
         prometheus::register_int_counter!(
             "mpc_num_passive_signature_requests_lookup_succeeded",
             "Number of passive signature requests successfully looked up in local DB"
         )
-        .unwrap();
-}
+        .unwrap()
+    });
 
-lazy_static! {
-    pub static ref MPC_OUTGOING_TRANSACTION_OUTCOMES: prometheus::IntCounterVec =
+pub static MPC_OUTGOING_TRANSACTION_OUTCOMES: LazyLock<prometheus::IntCounterVec> =
+    LazyLock::new(|| {
         prometheus::register_int_counter_vec!(
             "mpc_outgoing_transaction_outcomes",
             "Number of transactions sent by this node, by type and outcome",
             &["type", "outcome"],
         )
-        .unwrap();
-}
-
-lazy_static! {
-    pub static ref MPC_INDEXER_LATEST_BLOCK_HEIGHT: prometheus::IntGauge =
-        prometheus::register_int_gauge!(
-            "mpc_indexer_latest_block_height",
-            "Latest block height processed by the near indexer"
-        )
-        .unwrap();
-}
-
-lazy_static! {
-    pub static ref MPC_ACCESS_KEY_NONCE: prometheus::IntGauge = prometheus::register_int_gauge!(
+        .unwrap()
+    });
+pub static MPC_INDEXER_LATEST_BLOCK_HEIGHT: LazyLock<prometheus::IntGauge> = LazyLock::new(|| {
+    prometheus::register_int_gauge!(
+        "mpc_indexer_latest_block_height",
+        "Latest block height processed by the near indexer"
+    )
+    .unwrap()
+});
+#[allow(dead_code)]
+pub static MPC_ACCESS_KEY_NONCE: LazyLock<prometheus::IntGauge> = LazyLock::new(|| {
+    prometheus::register_int_gauge!(
```

Comment:
I got a complain from clippy here, it seems that we were not using that function and that was only uncovered by this change.